llama_index/llms/llama_cpp.py

```diff
@@ -37,6 +37,8 @@
 
 
 class LlamaCPP(CustomLLM):
+    DEFAULT_VERBOSITY = True
```

Comment:
should this be a part of the object class or just a constant above the class? (I feel like the latter makes a bit more sense)

Reply:
Sure!  Moved to the module level